### ISSUE TITLE
on Chocolatey

### ISSUE THREAD

**Issue by caglaryalcin** [Heroic-Games-Launcher/HeroicGamesLauncher#3965]
### Feature description

Heroic games launcher is available on winget. But I think it should also be available on chocolatey, which is as widely used as winget.

![image](https://github.com/user-attachments/assets/34dd8bce-3c25-476e-bec1-94113f608df6)

**Comment by CommandMC** [comment]
I don't really see a point in publishing to *another* database, which is inherently smaller than WinGet's (as WinGet is preinstalled on every Windows OS). Releases are already quite time-consuming.

That being said, if someone would create a PR to auto-update a potential Chocolatey version whenever we do a release (like it's currently done for Arch Linux's AUR), I don't see why that wouldn't get merged

**Comment by caglaryalcin** [comment]
I don't agree with you, Winget can give many different errors when installing applications. I can give you many examples of this. Also, chocolatey has hundreds of packages (known and unknown) and why shouldn't your app have them too? I don't think it should be that hard :)
